charts/karpenter/values.yaml

```diff
@@ -22,6 +22,8 @@ controller:
   clusterName: ""
   # -- Cluster endpoint
   clusterEndpoint: ""
+  # -- Default instance profile
+  defaultInstanceProfile: ""
```

Comment:
Thoughts on putting a note that clusterEndpoint, clusterName, and defaultInstanceProfile are all required?